iothub/device/src/DeviceClient.ConventionBasedOperations.cs

```diff
@@ -71,6 +71,34 @@ public Task<ClientProperties> GetClientPropertiesAsync(CancellationToken cancell
         /// </summary>
         /// <param name="callback">The callback to handle all writable property updates for the client.</param>
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        /// <example>
+        /// Inline:
+        /// <code language="csharp">
+        /// await client.SubscribeToWritablePropertyUpdateRequestsAsync(
+        /// 	async (writableProperties) =>
+        /// 	{
+        /// 		if (writableProperties.TryGetValue("targetTemperature", out WritableClientProperty targetTemperatureRequested))
+        /// 		{
+        /// 			var propertiesToBeUpdated = new ClientPropertyCollection();
```

Comment:
Suggest moving this above the `if`, and the call to `UpdateClientPropertiesAsync` after the `if`.